src/code42cli/cmds/alerts.py

```diff
@@ -159,14 +154,27 @@ def clear_checkpoint(state, checkpoint_name):
     "--or-query", is_flag=True, cls=searchopt.AdvancedQueryAndSavedSearchIncompatible
 )
 @opt.sdk_options()
+@click.option(
+    "--include-all",
+    default=False,
+    is_flag=True,
+    help="Display complete list of data items or only from top level of nested response.",
```

Comment:
Should the word `or` be `of`?

Reply:
Need more improvisation here, specifying "--include-all" does the other way around, it display all items and defaults to top level. 

How about `Display complete list of data items, defaults to data items from the first level of the nested response`

Reply:
@unparalleled-js 's verbiage suggestion works here, there is not a way to get any items from past the first level of the response except for with json/csv.